codegen/core/src/main/scala/com/lightbend/akkasls/codegen/Packaging.scala

```diff
@@ -32,6 +32,9 @@ case class FullyQualifiedName(
     parent: PackageNaming,
     descriptorObject: Option[FullyQualifiedName]) {
 
+  // FIXME: quick workaround for now. We may want to model this in a better way.
+  val isProtoMessage: Boolean = protoName.nonEmpty
```

Comment:
yeah, let make it more hacky. :-)

We can find a better solution for that, but I wanted to have a confirmation that the POJO idea can fly first.